apps-rendering/src/components/Layout/StandardLayout.tsx

```diff
@@ -134,6 +135,16 @@ const StandardLayout: FC<Props> = ({ item, children }) => {
 						<Metadata item={item} />
 						<Logo item={item} />
 					</section>
+
+					{item.design === ArticleDesign.Explainer &&
+						item.outline.length > 0 && (
```

Comment:
Will `outline` always exist as a property of item? If not I think we need to check outline exists first before checking the length. I think `item.outline?.length > 0` should work

Reply:
A great question. @JamieB-gu please correct me if I'm not explaining correctly. 
Item can be one of the following types:
```
type Item =
	| LiveBlog
	| DeadBlog
	| Review
	| Comment
	| Standard
	| Interactive
	| MatchReport
	| Letter
	| Obituary
	| Editorial
	| Correction
	| Interview
	| Analysis
	| Explainer;
```
And Explainer interface looks like this:
```
interface Explainer extends Fields {
	design: ArticleDesign.Explainer;
	body: Body;
	outline: Outline;
}
```

So in this case when we check `item.design === ArticleDesign.Explainer` we are sure that we're dealing with only the `Explainer` Item type and `outline` is a required field in this type.